tests/model_registry/conftest.py

```diff
@@ -482,3 +482,23 @@ def model_registry_rest_headers(current_client_token: str) -> dict[str, str]:
         "accept": "application/json",
         "Content-Type": "application/json",
     }
+
+
+@pytest.fixture(scope="class")
+def model_registry_deployment_containers(model_registry_namespace: str) -> list[dict[str, Any]]:
+    return Deployment(
+        name=MR_INSTANCE_NAME, namespace=model_registry_namespace, ensure_exists=True
+    ).instance.spec.template.spec.containers
+
+
+@pytest.fixture(scope="class")
+def model_registry_pod(admin_client: DynamicClient, model_registry_namespace: str) -> Pod:
+    mr_pod = list(
+        Pod.get(
+            dyn_client=admin_client,
+            namespace=model_registry_namespace,
+            label_selector=f"app={MR_INSTANCE_NAME}",
+        )
+    )
+    assert len(mr_pod) == 1
+    return mr_pod[0]
```

Comment:
there is already `model_registry_instance_pod` that I think is doing the same, could you please double check?

Reply:
I can't use model_registry_instance_pod as waits for the pods. I am assuming it needs the pods to be spinned up. In my case, I expect the pods to be there, if they are not, it is a bug.

Reply:
Resolving based on our conversation.